devsetup/scripts/standalone.sh

```diff
@@ -56,6 +56,12 @@ sudo dnf update -y
 EOF
 fi
 
+if [[ -e /run/systemd/resolve/resolv.conf ]]; then
+    HOST_PRIMARY_RESOLV_CONF_ENTRY=$(cat /run/systemd/resolve/resolv.conf | grep ^nameserver | grep -v '192.168' | head -n1)
```

Comment:
The variable will contain "nameserver <IP>" (a full line from resolv.conf) but i think in the code below we want just "<IP>" format. I think we should add something like `| awk '{ print $2; }'` at the end of the pipe.